src/main/java/io/github/jwdeveloper/spigot/tester/implementation/factory/TestClassModelFactory.java

```diff
@@ -80,7 +80,7 @@ private SpigotTest getSpigotTestInstance(Constructor constructor) throws Invocat
 
     private List<TestMethodModel> getTestMethods(Class<?> clazz) {
         var result = new ArrayList<TestMethodModel>();
-        var methods = Arrays.stream(clazz.getDeclaredMethods())
+        var methods = Arrays.stream(clazz.getMethods())
                 .filter(c -> c.isAnnotationPresent(Test.class))
```

Comment:
Whats the reason of this change? `getDeclaredMethod` also contains private/protected method I think tests should't be 
limited to be public only, If there is need to ignore method `Test` annotation has property `ignored`

Reply:
The reason I changed it is because getMethods takes methods from a super (abstract) class, while getDeclaredMethods doesn't. What exactly is the use case for private test methods?

Reply:
right, the Test framework I was using was allowed to make packet private method for tests, so some people could be used to this approach, I will later make code that collect all method even if there are in super class